pkgs/top-level/all-packages.nix

```diff
@@ -637,6 +637,11 @@ with pkgs;
 
   genymotion = callPackage ../development/mobile/genymotion { };
 
+  gams = callPackage ../tools/misc/gams {
+    license = config.gams.license;
```

Comment:
Still doesn't eval if the two config options aren't set (like on ofborg). You need to define safe defaults for both of them, as in ` license = config.gams.license or "someSensibleDefaultValue";`. Please verify that the package builds with these default values, in this case it should result in an unlicensed demo version using default options.

Reply:
Thanks for noticing, I added a option that if the variable should be undefined that its set to null
I was wondering where I should check if the var is defined right now im doing it in all-packages.nix since the chromium.nix is doing the same, but is that the right way to do it?